AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -23,6 +23,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+vertopal.com###vertonotif-container__bottom-right > div.vertonotif-content.vertonotif-type-warning > div.text
```

Comment:
I think that it can be simplified to:
```adblock
vertopal.com###vertonotif-container__bottom-right > .vertonotif-type-warning
```
and please move it to `! SECTION: Anti-adblock - Regular rules` section in https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Popups/sections/antiadblock.txt#L1864-L1869